test/check-machines-migrate

```diff
@@ -145,6 +147,7 @@ class TestMachinesMigration(VirtualMachinesCase):
         b.click("#migrate-button")
         b.wait_not_present("#migrate-modal")
         self.assertIn("subVmTest1", machine2.execute("virsh list --all"))
+        self.assertNotIn("subVmTest1", machine1.execute("virsh list --all"))
```

Comment:
Is this correct? When 'Run after migration' in unchecked the source VM will be left running I understand.

Reply:
Oh right, mistook the logic there, fixed it to `assertIn`